### ISSUE TITLE
demo to show notification service extension

### ISSUE THREAD

**Issue by RishabhTayal** [OneSignal/OneSignal-Cordova-Example#3]
Can we add an example to show how to use notification service extension with cordova and onesignal?

**Comment by Aarbel** [comment]
Yes could be great, right now its seems to require raw Xcode configurations, which is not compatible with the way we work with Cordova.
